client/my-sites/stats/info-panel/index.jsx

```diff
@@ -188,7 +188,7 @@ class StatsInfoPanel extends React.PureComponent {
 								>
 									<Gridicon icon="help-outline" />
 									{ this.props.translate( 'How do I get more visitors?', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'How do I get more visitors?', { context: 'Stats: Posts & Pages info box documentation link'} )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)


:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?